### DISCUSSION TITLE
`grid-flow-col` behaving strangely

### DISCUSSION THREAD

**Discussion by jayrdi** [statamic/cms#6076] [Q&A]
I am following the Statamic tutorial collection on YouTube and have created some placeholder blogs, with this index page:

![image](https://user-images.githubusercontent.com/7912545/169836465-0aa28686-81a4-4cd7-86c5-61c775442dc4.png)

When I navigate to the page though, I see that they only display on a single row, and that `grid-cols-2` is making the first two blogs 0 width:

![image](https://user-images.githubusercontent.com/7912545/169837173-e6c19843-28eb-49ca-b062-47849ed4a6c8.png)

If I change to `grid-cols-3`, I get this:

![image](https://user-images.githubusercontent.com/7912545/169837783-ee3994ac-200f-4b8a-8a3d-a0650bc50c48.png)

Can anyone explain what is happening here?

Thanks

**Comment by duncanmcclean** [comment]
Are you running Laravel Mix? `npm run watch`

**Reply by jayrdi** [reply]
Yes, running Laravel Mix and using `npm run watch`. Also tried clearing cache (as that had been an issue before).

**Comment by austriker27** [comment]
Does it work if you drop that code into https://play.tailwindcss.com/? I'm guessing its something with your local build process because at first look the tailwind looks right.

**Reply by jayrdi** [reply]
"play.tailwind" seems to be down; I can't access it. Anyway, someone on Reddit suggested removing `grid-flow-cols` as they thought it was unnecessary to have as well as `grid-cols-2`. It did the trick. Maybe something changed in the way that the browser interprets that since the tutorial was made?

**Reply by jasonvarga** [reply]
[play.tailwindcss.com](https://play.tailwindcss.com/)

**Reply by jayrdi** [reply]
Ah, that's very handy, thanks. I'll bookmark that!
